index.js

```diff
@@ -371,10 +371,15 @@ module.exports = function (config, deps) {
         .set(common.SESSION_TOKEN_HEADER, user.getUserToken())
         .end(
         function (err, res) {
+
           if (err != null) {
             return cb(err);
+          } else if (res.status !== 200) {
+            return cb(res.body);
```

Comment:
there was a patter of "this is the status we expect and treat others as though something has gone wrong"